app/views/shared/_one_time_code_input.html.erb

```diff
@@ -17,10 +17,12 @@ classes = ['field monospace usa-input one-time-code-input']
 classes << local_assigns.delete(:class) if local_assigns[:class]
 %>
 
+
 <%= text_field_tag(
   name,
   nil,
   'aria-invalid': 'false',
+  'aria-label': 'code',
```

Comment:
This will override the `<label>One-time security code</label>` label. Do we want that? If we do, we should translate this too, since it's announced to assistive technology.